python/cudf_polars/cudf_polars/experimental/benchmarks/utils_new_frontends.py

```
@@ -814,9 +812,15 @@ class QueryResult:
     sort_keys: list[tuple[pl.Expr, bool]] | None = None
 
 
-def _collect_statistics(engine: StreamingEngine | None) -> dict[str, Any] | None:
+def _collect_statistics(engine: pl.GPUEngine | None) -> dict[str, Any] | None:
     """Gather + clear per-rank rapidsmpf statistics into a merged dict."""
-    return None if engine is None else engine.global_statistics(clear=True).to_dict()
+    if engine is None:
+        return None
+    from cudf_polars.experimental.rapidsmpf.frontend.core import StreamingEngine
```

Ruff (0.15.12)

[warning] `import` should be at the top-level of a file

(PLC0415)